tools/dgeni/src/transforms/daffodil-guides-package/helpers/navigation-trie.ts

```diff
@@ -117,7 +117,7 @@ export class NavigationTrie {
     else if (child && child.children.length == 0) {
       const node = new NavigationTrie('', 'Overview', child.path);
       delete (child.path);
-      child.appendChild(node);
+			child.appendChild(node);
```

Comment:
Doesn't need changed.

Reply:
I think I can only fix this with a squash commit, because git is just being weird with its detection of changes. I can do this after you add a "lgtm" and before you approve.